src/main/perl/RuleBasedEditor.pm

```diff
@@ -148,27 +152,27 @@ end of the line if it is modified by L<CAF::RuleBasedEditor>.
 
 =item *
 
-LINE_FORMAT_KEY_VAL:        key val (e.g. Xrootd, Apache)
+LINE_FORMAT_KW_VAL:        key val (e.g. Xrootd, Apache)
```

Comment:
apologies, with elsewhere i meant outside of quattor. i don't mind the exact name that much, but after the 16.4 release it should change anymore (or changes would require backwards compatible code).